installation/install.sh

```
@@ -960,6 +960,7 @@ check_system(){
     else
         cs_check_hostname_zstack
     fi
+    [ x`systemctl is-enabled cockpit.socket 2>/dev/null` = x"enabled" ] && (systemctl disable cockpit.socket && systemctl stop cockpit.socket)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)